### DISCUSSION TITLE
How to clear all bruteforce entries in the database

### DISCUSSION THREAD

**Discussion by jakobroehrl** [nextcloud/all-in-one#3381] [Questions] [answered]
I want to show and clear all my bruteforce entries. 
This is possible within the database directly: https://help.nextcloud.com/t/solved-bruteforce-detection-blocking-my-ip-but-theres-no-oc-bruteforce-attempts-database/7652

How can I use psql?

**Comment by szaimen** [comment, marked as answer]
Hi, based on https://github.com/nextcloud/all-in-one#bruteforce-protection-faq you might try `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ security:bruteforce:reset 0.0.0.0`

**Comment by jrstrunk** [comment]
It seems that any IP I log in from is blocked. Is there a way I can execute SQL statements on the PostgreSQL database to inspect the oc_bruteforce_attempts table?

Edit: Never mind I figured out the command `sudo docker exec -it nextcloud-aio-database psql -U oc_nextcloud -d nextcloud_database` will let me into the postgresql database. However, I do not see any entries into the oc_bruteforce_attempts table but I keep getting the "Too many requests" error on fresh install :(

**Reply by maste9** [reply]
I experience exactly the same phenomenon.
I made a fresh docker-compose based install behind a nginx proxy manager. After I finally managed to reach my domain I followed the instructions and all containers were shown up. I opened the login page and ... it appears including the information that there have been too many login attempts from that ip address.

I waited far longer than the proclaimed 30sec multiple times but never got unblocked.

I then tried to reset the several ips (internal, external and so on ...) I could imagine but that didn't change anything. I shutdown all my devices with any clients ... nothing changed ever.
Finally I checked the database itself and found out that the table 'oc_bruteforce_attempts' didn't contain any entries ... So the blockage isn't really caused by the bruteforce protection, it seems but there's kind of wrong failure message about what is excluding anybody from the fresh install.

I will return now to my old fashioned setup and lay down any attempt of migration to AIO until there's any hint what might have gone wrong. I would give you info from the logs ... if there were any. Any container log I checked looked normal and I can't find the info I usually find in nextcloud.log  ... like login trials or sth.

@jrstrunk  I guess you didn't make any progress since your last post either? :confused:

**Reply by jrstrunk** [reply]
@maste9 I was able to solve the issue by adding my Caddy reverse proxy's internal IP address (172.18.0.1) to the trusted proxies list as discussed in #3396 and then waiting a couple hours before trying again. Hope this helps!

**Reply by maste9** [reply]
Thank you @jrstrunk !

Actually I couldn't let it lie and had a little nightshift :grin: 

I had the idea that my data-dir that I  added to the docker-compose configuration wasn't empty which has broken the install. So I just followed the instructions to reset everything, deleted all content from the data-dir and started again from scratch with the [migration](https://github.com/nextcloud/all-in-one/blob/main/migration.md).

I haven't been locked out again but also needed to config the trusted_proxies in the config.php as too many clients connecting by its ip throttled down the account.

So now I have a successfully migrated cloud and am quite impressed by it. The only thing I have to clarify is now why database, nextcloud and redis containers are causing 20-50% CPU continuously  even if none of the three accounts is actively using the cloud :confused:  But that's maybe for another day :yawning_face:
